app/js/mms/directives/veMenu.js

```diff
@@ -123,6 +127,19 @@ function veMenu(CacheService, $state, $templateCache, $sce) {
             var liWidth = (eltWidth * 0.65)/crumbcount;
             scope.truncateStyle={'max-width': liWidth, 'white-space': 'nowrap', 'overflow': 'hidden', 'text-overflow': 'ellipsis', 'display': 'inline-block'};
         }
+
+        function getHrefForProject(project) {
```

Comment:
you can just make these scope functions - using scope.functionname

Reply:
you meant sth like this: scope.getHrefForProject = function() {}.....?
It is a better practice put fn definition at the bottom and the interface at the top so that at a glance, we can tell what we can do with this directive.

Reply:
i know... just wanted to be consistent but it's fine. i'll merge